Builds breaking sometimes, subtly, again

#1022 seemed to fix the issue, but hasn't. @pscott31 is still seeing it, and I can recreate it.

# To recreate
```bash
rm -rf node_modules && nvm install 16.14.0 && yarn && ./node_modules/.bin/nx run trading:build:production --source-map=false && cd dist/apps/trading && python3 -m http.server 8001
```

Output:
```
   ✖    1/4 dependent project tasks failed (see below)
   ✔    3/4 dependent project tasks succeeded [2 read from cache]

 ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————


> nx run react-helpers:build

Bundling react-helpers...
Error during bundle: Error: /Users/edd/src/vega/frontend-monorepo/libs/react-helpers/src/lib/generic-data-provider.ts(10,58): semantic error TS6059: File '/Users/edd/src/vega/frontend-monorepo/libs/types/src/index.ts' is not under 'rootDir' '/Users/edd/src/vega/frontend-monorepo/libs/react-helpers/src'. 'rootDir' is expected to contain all source files.
Error during bundle: Error: /Users/edd/src/vega/frontend-monorepo/libs/react-helpers/src/lib/generic-data-provider.ts(10,58): semantic error TS6059: File '/Users/edd/src/vega/frontend-monorepo/libs/types/src/index.ts' is not under 'rootDir' '/Users/edd/src/vega/frontend-monorepo/libs/react-helpers/src'. 'rootDir' is expected to contain all source files.
Bundle failed: react-helpers

 ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

 >  NX   Ran target build for project trading and 19 task(s) it depends on (8s)
 
         With additional flags:
           --sourceMap=false
 
    ✖    1/4 failed
    ✔    3/4 succeeded [2 read from cache]
 

```

--- a/nx.json
+++ b/nx.json
@@ -1,5 +1,5 @@
 {
-  "npmScope": "@vegaprotocol",
+  "npmScope": "vegaprotocol",
   "affected": {
     "defaultBase": "master"
   },
@@ -18,7 +18,16 @@
       "runner": "@nrwl/nx-cloud",
       "options": {
         "cacheableOperations": ["build", "lint", "test", "e2e"],
-        "accessToken": "OTY4ZjdlZTItNGIwNy00NDcyLTllZjctOWIzYTg1NWE0Yzg1fHJlYWQtd3JpdGU="
+        "accessToken": "OTY4ZjdlZTItNGIwNy00NDcyLTllZjctOWIzYTg1NWE0Yzg1fHJlYWQtd3JpdGU=",
+        "runtimeCacheInputs": [
+          "echo $NX_VEGA_NETWORKS",
+          "echo $NX_VEGA_ENV",
+          "echo $NX_VEGA_URL",
+          "echo $NX_TENDERMINT_URL",
+          "echo $NX_TENDERMINT_WEBSOCKET_URL",
+          "echo $NX_USE_ENV_OVERRIDES",
+          "echo $NX_ETHEREUM_PROVIDER_URL"
+        ]
       }
     }
   },

